src/components/Modals/NewCourse/NewCourseModal.vue

```diff
@@ -191,17 +192,16 @@ export default Vue.extend({
       this.reset();
       this.closeCurrentModal();
     },
-    onSelectedChange(selected: readonly string[]) {
-      this.selectedRequirementIDs = selected;
+    onSelectedChange(selected: string) {
+      this.selectedRequirementID = selected;
     },
     reset() {
```

Comment:
Should we also reset `this.requirementsThatAllowDoubleCounting = []`?

Reply:
No requirements that allow double counting is immutable. A requirement either allows or doesn't allow double counting, regardless of user choices